### ISSUE TITLE
Doesn't work with Sortable.js

### ISSUE THREAD

**Issue by david-stefan** [andywer/drag-mock#12]

**Comment by jeansymolanza** [comment]
Have you managed to find an alternative for sortable.js?

**Comment by david-stefan** [comment]
> Have you managed to find an alternative for sortable.js?

No, not really. In fact, I had to implement an additional, fallback way of sorting elements by mouse-clicks on top of drag-and-drop, just to be able to run and pass several end-to-end tests. Also, I don't think changing the library would help, since it's a ChromeDriver issue as far is I've discovered.
